staging/src/github.com/openshift/oc/pkg/helpers/graph/genericgraph/test/new-project-deployed-app.yaml

```diff
@@ -543,7 +543,7 @@ items:
       name: frontend
     tls:
       insecureEdgeTerminationPolicy: Redirect
-      type: Edge
+      termination: edge
```

Comment:
are you sure about lowercase?

Reply:
Yeah, double checked with API, it's not the type I care here but the termination. Previously we relied on defaulting to kick in, now these tests need to get object as if it came from the server.